Code/Tools/ProjectManager/Source/ProjectManagerWindow.cpp

```diff
@@ -34,6 +34,7 @@ namespace O3DE::ProjectManager
         QVector<ProjectManagerScreen> screenEnums =
         {
             ProjectManagerScreen::Projects,
+            ProjectManagerScreen::CreateGem,
```

Comment:
Does adding `CreateGem` as the second entry in `screenEnums` affect the screen stack widget initialization in `ScreenCtrl.BuildScreens`?

I think this may be fine though, since iirc each tab is first checked to see if it is meant to go to `ScreenCtrl.m_tabWidget` before adding it to that data structure. But otherwise unsure if reordering has unintended consequences.